### ISSUE TITLE
AppImageAssistant -t has issues with rpm

### ISSUE THREAD

**Issue by probonopd** [probonopd/AppImageKit#54]
AppImageAssistant -t has issues with rpm: When trying to install packages, we can sometimes get

```
error: unpacking of archive failed on file (...): cpio: chown
```
Maybe it is related to
https://features.opensuse.org/312718

> librpm cannot deal with updating the attributes of a directory-type inode that has been hidden by a new directory-type inode of another vfsmount

**Comment by probonopd** [comment]
Somehow this issue seems to be bigger on some host systems than on others. I am not 100% sure but I think it used to work better for me on another machine than it does today.

**Comment by probonopd** [comment]
I don't seem to get the error anymore when using [OverlayFS](https://github.com/probonopd/AppImageKit/pull/122); others please test as well.

**Comment by probonopd** [comment]
AppImageAssistant hast been replaced by `appimagetool`. Use it instead. Need to move `testappimage` over to it.

**Comment by TheAssassin** [comment]
Is this really old issue still relevant? I think we can close it.

**Comment by probonopd** [comment]
No, the `testappimage` functionality has not been put into `appimagetool` yet, but it should.
